packages/sdk/src/assets/bridge-asset.ts

```diff
@@ -0,0 +1,267 @@
+    } else {
+      // we call jobEstimator on destination chain
+      // by supplying 10 million gas, we get back result of how much gas is left from simulation
+      // subtract leftover gas from 10 million to know exactly how much gas is used for tx
+
+      const holographToChainId = getNetworkByChainId(toChainId).holographId
+      const gasEstimationHelperAddress: Address = getGasEstimationAddress(toChainId)
+      const maxGasValue: bigint = hexToBigInt(('0x' + 'ff'.repeat(32)) as Hex)
+      const gasLimitStub: bigint = getTestGasLimit(toChainId) // this is a very big gas value so we can use to estimate with
+
+      const bridgeRequestPayloadStub = this._createBridgeOutPayload(
+        chainId,
+        holographToChainId,
+        contractAddress,
+        maxGasValue,
+        maxGasValue,
+        bridgeOutPayload,
+      )
+
+      const consumedGas = BigInt(
+        (
+          await this._operator.simulateContractFunction({
+            chainId,
+            functionName: 'jobEstimator',
+            args: [bridgeRequestPayloadStub],
+            options: {gas: gasLimitStub}, //{account: gasEstimationHelperAddress , gas: TEST_GAS_LIMIT},
+          })
+        ).result,
+      )
+
+      destinationGasLimit = gasLimitStub - consumedGas //TODO: ver se dá para chamar o gasEstimate ao invés de fazer essa conta
+    }
+
+    if (gasController?.gasLimitMultiplier) {
+      destinationGasLimit = (destinationGasLimit * BigInt(gasController.gasLimitMultiplier)) / BigInt(100)
+    }
+
+    if (gasController?.gasPrice) {
+      destinationGasPrice = gasController.gasPrice
+    } else {
+      // we extract the most recent and most accurate gas prices for destination network
+      const provider = this._providers.byChainId(chainId)
+      const gasPricing: GasPricing = await initializeGasPricing(provider)
+
+      // get gas price based on eip-1559 support
+      destinationGasPrice = gasPricing.isEip1559 ? gasPricing.maxFeePerGas! : gasPricing.gasPrice!
+    }
+
+    if (gasController?.gasPriceMultiplier) {
+      destinationGasPrice = (destinationGasPrice * BigInt(gasController.gasPriceMultiplier!)) / BigInt(100)
+    }
+
+    return {
+      destinationGasPrice,
+      destinationGasLimit,
+    }
+  }
+
+  async estimateBridgeOut(
+    chainId: number,
+    toChainId: number,
+    contractAddress: Address,
+    bridgeOutPayload: Hex,
+  ): Promise<EstimateBridgeOutResult> {
+    const {destinationGasPrice, destinationGasLimit} = await this._estimateBridgeOutDestinationGas(
+      chainId,
+      toChainId,
+      contractAddress,
+      bridgeOutPayload,
+    )
+
+    const holographToChainId = getNetworkByChainId(toChainId).holographId
+
+    // now that we have gasLimit and gasPrice, we update the payload to include the proper data needed for estimating fees
+    const bridgeRequestPayload = (
+      await this._createBridgeOutPayload(
+        chainId,
+        holographToChainId,
```

Comment:
the name makes me think that its doing a conversion. Maybe  `destinationHolographChainId`. that way it keeps withe same form as the other destination variables.

Reply:
good idea